Website/index.html

```diff
@@ -114,6 +114,21 @@ <h2>Contact Us</h2>
 			<a href="mailto:saptarshi.programmer@gmail.com" type="button"><i class="fa-sharp fa-solid fa-envelope"></i>
 				Email</a>
 		</div>
+		<div class="contact-me" >
+			<section class="contact section" id="contact">
+				<br>
+				<form action="" method="POST">
+					<input type="hidden" name="_subject" value="New Query Arrived">
+					<input name="name" type="text" class="feedback-input" placeholder="Name" required />
+					<input type="hidden" name="_next" value="http://127.0.0.1:5500/thankyou.html">
```

Comment:
I think the below change is also required and `thankyou.html` needs to be created.
```suggestion
					<input type="hidden" name="_next" value="https://saptarshisarkar12.github.io/Drifty/thankyou.html">
```

Reply:
Yes check this.

Reply:
> > @SaptarshiSarkar12 Yes, I forgot to mention that you have to just add your mail action tag e.g: action = "mailto:yourmailid".
> 
> Please add that

Ok adding

Reply:
saptarshi.programmer@gmail.com

Reply:
> 

No that's not mandatory

Reply:
> > 
> 
> No that's not mandatory

Which one?

Reply:
> > > 
> > 
> > 
> > No that's not mandatory
> 
> Which one?

@KIET7UKE Please tell.